stout/grpc/client.h

```diff
@@ -149,21 +149,41 @@ struct _Call {
 
                 RequestType_* request = nullptr;
                 ::grpc::WriteOptions* options = nullptr;
-                bool finish = false;
+
+                enum NextAction { NOTHING,
```

Comment:
I suggest using `enum class` declaration instead. `enum` is an int and (too) easily casts to it while an `enum class` is a proper type.